.buildkite/pipeline.yml

```diff
@@ -2,11 +2,11 @@
 common_params:
   # Common plugin settings to use with the `plugins` key.
   - &common_plugins
-    - automattic/a8c-ci-toolkit#3.1.0
+    - $CI_TOOLKIT_PLUGIN
```

Comment:
Personal take maybe, but at this point now that we have `$CI_TOOLKIT_PLUGIN` available to DRY that value, I'm not sure it's worth keeping the `&common_plugins` anchor—and still use `plugins: *common_plugins` in each step—instead of have each step now directly use `plugins: [$CI_TOOLKIT_PLUGIN]`?

Reply:
The thought crossed my mind. I was too lazy to implement it at the time.

One thing in favor of keeping the current approach is that we'd be able to add `n` new plugins to all steps with a one line change. But:

1. YAGNI
2. Reintroducing the DRY abstraction would be straightforward if needed.